assessments/0009-in-toto/in-toto-implementation.md

```diff
@@ -0,0 +1,142 @@
+| **Project owners** define the layout to be followed when, e.g., using the in-toto CLI tools. When doing so, they specify who is intended to sign for every piece of link metadata, any sublayouts that may exist, and how to further verify accompanying metadata. | New users need an overview, demos, templates, and basic instructions. Experienced users need deeper architectural info, use cases, and examples. |
+| **Functionaries** perform the intended actions and produce link metadata for each step. | *?? do these users read in-toto doc at all, or is the project owner responsible for instructing them in how to sign and verify their steps??* |
+| **Contributors** can be contributors to the *code* base and to the *documentation* of the OSS project itself. | Contributors need project standards and instructions, community guidelines, and GitHub instructions specific to the project. This information is often in the repo, and not in the project or product documentation.  |
+| **Code contributors**  are members of the community who: *make code changes*, *submit changes to specifications*, *create new integrations*, and *submit issues, feature requests and more* | These contributors need to understand the GitHub repo structure, the contribution policies and procedures, coding and naming standards and conventions. ITE developers and integrators need direction on how and where to document their own additions. |
+| **Doc contributors**  (to be added) are members of the community who use the **Edit** button to make changes to published documents, or create new documentation. | These contributors need clear doc contribution standards, naming and style conventions, and clear instructions on how to create or edit existing doc, and where documents should go (that is, what should go on the website or in GitHub, where a subject fits in the doc architecture) |
+
+ 
+
+## General doc plan
+To begin achieving the goal of documentation that meets the needs of different user populations and is discoverable by the intended readers, we recommend the following general plan.
+
+1. Initial tasks:
+
+   a. Create a Documentation home page on the [project web site](https://in-toto.io), linked prominently from the About menu. 
+
+   b. Create a **Getting Started** page on the web site from the existing README content in the [main repo](https://github.com/in-toto/in-toto.README.md).
+
+   c. Link **Getting Started** as the first menu item in the **Get started** menu (currently the first item is a link to a demo).
+
+   d. Expose parts of the product specification as separate named documents on the website, as:
+     - [System Overview](https://github.com/in-toto/docs/blob/master/in-toto-spec.md#2-system-overview)  (compare content to https://in-toto.io/in-toto/README and current website About - create versions of increasing depth to address to specific audiences)
```

Comment:
@dwelsch-esi the `https://in-toto.io/in-toto/README` link is invalid. What was it meant to be?

/cc @nate-double-u

Reply:
I would guess https://github.com/in-toto/in-toto/blob/develop/README.md, but i think https://in-toto.io/in-toto/ would be more appropriate now.

Reply:
or maybe @jbogarthyde was suggesting comparing and bringing the one in line with the other?